### ISSUE TITLE
Obtaining STEAD data

### ISSUE THREAD

**Issue by yetinam** [smousavi05/STEAD#5]
Hey,

I'm having trouble obtaining the STEAD dataset. I downloaded the files from GDrive and tried unpacking them with both unzip and 7zip, which both failed, claiming the zip file was corrupted. I tried unpacking the archive on two different linux machines and tried both the full dataset and the first chunk. I pasted the output from unzip below.

I tried downloading the uncompressed files directly from mega, but got a warning that the file size exceeds the free download quota. While there are no clear numbers from mega, [this site](https://www.cloudstorageoptions.com/mega/) suggests that the quota is around 1 GB per 6 hours.

Therefore I was not able to obtain the STEAD dataset. Do you have any suggestion how I might be able to obtain the STEAD dataset? Or maybe which techniques I could try to unpack the zip archive?

Thanks!

```
 $ unzip chunk1.zip 
Archive:  chunk1.zip
warning [chunk1.zip]:  12884901888 extra bytes at beginning or within zipfile
  (attempting to process anyway)
file #1:  bad zipfile offset (local header sig):  12884901888
  (attempting to re-compensate)
  inflating: chunk1.hdf5             
  error:  invalid compressed data to inflate
```

**Comment by smousavi05** [comment]
Hi, 

For the unzipping problem: the zipped files have been tested and used by several people, so the files should be okay but I'll double-check again. However, I recommend after making sure that the zip files have been downloaded completely (their size should be the same as those listed on the Github page) try to unzip them on a Mac machine. 

For downloading from Mega: the bandwidth problem can not be from our end. Data is on a paid account with very high bandwidth. Have you tried using their app for data transfer? also check one of the closed issues here, I remember one of the fellow seismologists posted something about it. 

I'll look more closely into these issues and also try to upload an unzipped version to Gdrive as well. As I do not have access to high-speed internet at moment, this may take a few days. I'll get back to your layer.

**Comment by yetinam** [comment]
Hey,
thank you for the very quick reply. Indeed a colleague of mine was able to unpack the zip archives on Mac OS without any issues. By comparing md5sums we checked that his and my version of the zip archive are indeed identical.

I look a bit into the problem and it seems to be a certain incompatibility between the Mac OS zip implementation and the zip implementation of Linux (or at least the linux versions I use) for large archives. It also affects 7zip. There is some explanation in [this discussion thread](https://unix.stackexchange.com/questions/438368/unix-unzip-is-failing-but-mac-archive-utility-works), in particular in the last answer.

In addition I also managed downloading the files from Mega, using [MegaCMD](https://github.com/meganz/MEGAcmd). Thanks for the pointer to the prior issue here.

I'm closing the issue, as I now have access to the STEAD dataset. If you want to have a further look into the unzip issue on Linux, feel free to reach out and I'll try providing you further information.
